plugins/da/qml/Keyboard_da_tablet.qml

```diff
@@ -18,8 +20,11 @@ import QtQuick 2.0
 import keys 1.0
 
 KeyPad {
+    id: keypadRoot
+
     content: c1
     symbols: "languages/Keyboard_symbols_tablet.qml"
+    alternativeLayouts: [ "Dvorak" ] // list of alternative layouts, like Dvorak, Bepo, Splitted...
```

Comment:
I don't know if you noticed, but this has to be associated with an "authorization" for the layout alternative in KeyboardContainer.qml in the function alternativeLayoutIsSupported. After a second thought, the need for that function is not really obvious, maybe I should just remove it in the future.